django/template/base.py

```diff
@@ -117,6 +117,32 @@ def __str__(self):
         return self.msg % self.params
 
 
+class UndefinedVariable:
+    """
+    This is the result of evaluating a template expression that
+    contains an undefined variable.
+
+    An instance of this class evaluates to False in a boolean
+    context. All instances are treated as being equal to each other.
+    """
+
+    def __init__(self, var_rep=""):
```

Comment:
I think `var_repr` is more explanatory than just `rep`. Also, use single quotes rather than double quotes unless the string contains a single quote.